selfstat/selfstat.go

```diff
@@ -17,7 +17,7 @@ import (
 )
 
 var (
-	registry *rgstry
+	registry *Registry
```

Comment:
This change should be ok as `registry Registry` differs in case (unlike the earlier `TimeSeries` which didn't...)